app/src/main/java/io/neurolab/About_Us.java

```diff
@@ -2,12 +2,27 @@
 
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.view.View;
+import mehdi.sakout.aboutpage.AboutPage;
 
 public class About_Us extends AppCompatActivity {
```

Comment:
Change to camel case - AboutUsActivity

Reply:
@iamareebjamal did the changes. I was about to do this earlier only but there was a pending PR [here](https://github.com/fossasia/neurolab-android/pull/144) regarding the same issue. However, I did the changes in my PR itself as 144 has merge conflicts and the PR is kept stagnant since quite a few days with no updates from the contributor. Thanks.